src/promptflow/tests/sdk_cli_azure_test/e2etests/test_connection_operations.py

```diff
@@ -1,25 +1,30 @@
 # ---------------------------------------------------------
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # ---------------------------------------------------------
+
 import pydash
 import pytest
 
 from promptflow._sdk.entities._connection import _Connection
+from promptflow.azure import PFClient
 from promptflow.azure._restclient.flow_service_caller import FlowRequestException
+from promptflow.azure.operations._connection_operations import ConnectionOperations
 from promptflow.connections import AzureOpenAIConnection, CustomConnection
 from promptflow.contracts.types import Secret
 
+from .._recording_base import PFAzureIntegrationTestCase
+from .._recording_utils import fixture_provider
 
-@pytest.fixture
-def connection_ops(ml_client):
-    from promptflow.azure import PFClient
 
-    pf = PFClient(ml_client=ml_client)
-    yield pf._connections
+@pytest.fixture(scope="class")
+def connection_ops(request: pytest.FixtureRequest, pf: PFClient) -> ConnectionOperations:
```

Comment:
seems this fixture appears in multiple files? How about put them in the conftest